snips/config.json

```diff
@@ -36,5 +36,5 @@
     "speaker": "str",
     "assistant": "str"
   },
-  "image": "homeassistant/{arch}-addon-snips"
+  "image": "tschmidty/{arch}-addon-snips"
```

Comment:
We should always publish all images under our account.

Reply:
I don't have access to that account, but if it allows shared access I'd be happy to publish it. My docker id is 'tschmidty'

Reply:
Pascal publishes all the add-ons based on the source code.